c2corg_api/views/user.py

```diff
@@ -108,8 +111,9 @@ def __init__(self, request):
         partial(validate_unique_attribute, "username")])
     def post(self):
         user = schema_create_user.objectify(self.request.validated)
+        user.forum_username = user.username  # same usernames by default
         user.password = self.request.validated['password']
-        user.update_validation_nonce(VALIDATION_EXPIRE_DAYS)
+        user.update_validation_nonce('regemail', VALIDATION_EXPIRE_DAYS)
```

Comment:
I clearly saind constants are the way to go in the general case.
Typically, the expiration time should be a constant and its value easily modifiable in one place.

The values used for `purpose` need to match, in the same way the attribute name need to match the one used during validation. Not sure using constants everywhere, in all cases, would be very pythonic.